crates/ruff_linter/src/rules/flake8_bugbear/rules/mutable_contextvar_default.rs

```diff
@@ -96,7 +97,7 @@ pub(crate) fn mutable_contextvar_default(checker: &mut Checker, call: &ast::Expr
                     && !is_immutable_func(func, checker.semantic(), &extend_immutable_calls)))
         && checker
             .semantic()
-            .resolve_qualified_name(&call.func)
+            .resolve_qualified_name(map_subscript(&call.func))
```

Comment:
It's unrelated to this PR, shoud we check if `call.func` first before checkking if `arguments` has a keyword `default`?